apps/firebase/src/celo-adapter.ts

```diff
@@ -137,99 +137,76 @@ export class CeloAdapter {
     to: string,
     amount: string,
     alwaysTransfer: boolean = false,
-  ) {
+  ): Promise<{ [key: string]: CeloTransactionObject<boolean> }> {
     const mento = this.mento
     if (!mento && this.useMento) {
       throw new Error('Must call init() first')
     }
     const celoToken = await this.kit.contracts.getGoldToken()
 
-    return this.kit.celoTokens.forStableCeloToken(
-      async (info: StableTokenInfo) => {
-        const token = await this.kit.celoTokens.getWrapper(
-          info.symbol as StableToken,
-        )
-        const [faucetBalance, recipientBalance] = await Promise.all([
-          token.balanceOf(this.defaultAddress),
-          token.balanceOf(to),
-        ])
-
-        const stableTokenAddr = token.address
+    return Object.fromEntries(
+      await Promise.all(
+        Object.keys(StableToken).map(async (symbol) => {
+          const token = await this.kit.celoTokens.getWrapper(
+            symbol as StableToken,
+          )
+          const [faucetBalance, recipientBalance] = await Promise.all([
+            token.balanceOf(this.defaultAddress),
+            token.balanceOf(to),
+          ])
 
-        const realAmount = this.fadeOutAmount(
-          recipientBalance,
-          amount,
-          alwaysTransfer,
-        )
+          const stableTokenAddr = token.address
 
-        if (realAmount.eq(0)) {
-          console.info(
-            `skipping ${
-              info.symbol
-            } for ${to} balance already ${recipientBalance.toString()}`,
+          const realAmount = this.fadeOutAmount(
+            recipientBalance,
+            amount,
+            alwaysTransfer,
           )
-          return false
-        }
-        console.info(
-          `sending ${to} ${realAmount.toString()}${
-            info.symbol
-          }. Balance ${recipientBalance.toString()}`,
-        )
 
-        if (faucetBalance.isLessThanOrEqualTo(realAmount)) {
-          if (mento) {
-            const quoteAmountIn = await mento.getAmountIn(
-              celoToken.address,
-              stableTokenAddr,
-              realAmount.toString(),
-            )
+          if (realAmount.eq(0)) {
             console.info(
-              `swap quote ${quoteAmountIn.toString()} for ${realAmount.toString()} `,
-            )
-            const maxCeloToTrade = quoteAmountIn.div(100).mul(103).toString() // 3% slippage
-            await this.increaseAllowanceIfNeeded(
-              new BigNumber(maxCeloToTrade),
-              info,
+              `skipping ${symbol} for ${to} balance already ${recipientBalance.toString()}`,
             )
 
-            const swapTxObj = await mento.swapOut(
-              celoToken.address,
-              stableTokenAddr,
-              realAmount.toString(),
-              maxCeloToTrade.toString(),
-            )
-            console.info('swap TX', swapTxObj)
-            await this.signer.sendTransaction(swapTxObj)
-          } else {
```

Comment:
probably makes sense to remove the related `if mento` tooo

Reply:
> ## Deployment
> I propose we merge and deploy this PR as a partial fix.
> 
> That way we unblock developers that wish to faucet cStables. At the moment, the faucet only makes CELO transfers and devs have to swap or mint at [app.mento.org](https://app.mento.org/).
> 
> On that basis I'll merge this PR and deploy the firebase app to production.
> 
> ## Out-of-scope for this PR
> Then separately, make a new PR with:
> 
> * this [refactor proposed](https://github.com/celo-org/faucet/pull/156#discussion_r1473096120) by @aaronmgdr
> * any other changes that might be necessary to improve cStable disbursements.

Source: [comment in this PR](https://github.com/celo-org/faucet/pull/156#issuecomment-1924515888)